_includes/layouts/post.html

```diff
@@ -18,7 +18,7 @@
         {% if tags[1] %}
         <a href="{{ '/' | url }}{{ tags[1] }}" class="text-bold" rel="homepage">{{tags[1]}}</a>,
```

Comment:
Missed this one?

Reply:
Yeah that one won't compile because it's an array. Tried a few different notations then left it later.

Reply:
Oh, okay.  Not a big deal, I guess just trying to be as consistent as possible..